src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -2297,6 +2297,36 @@ void ObjectMgr::LoadCreatures()
     LOG_INFO("server.loading", " ");
 }
 
+void ObjectMgr::LoadCreatureSparring()
```

Comment:
could this table use ingame reloading? if so add support for it

Reply:
I will look to add reloading support.

Reply:
It is easy if you load them into store at startup

Reply:
But I can't reload all creature IG. At least not with the current system. It's easy if we don't store the sparring pct as variable and if acces directory to the store. The only worse thing is if thé store start to be really, you will iterate very often the store and the CPU could  drasticly increase.